<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageMeasure</class>
 <widget class="QWidget" name="PageMeasure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonZoomIn_8">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton
{

	border-style: flat;
	font: 12px;
	color: #DEDEDE;	
}


QToolButton:checked
{
	color: #0DAF9D;
}

</string>
       </property>
       <property name="text">
        <string>模型</string>
       </property>
       <property name="iconSize">
        <size>
         <width>39</width>
         <height>39</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="popupMode">
        <enum>QToolButton::DelayedPopup</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonZoomIn_9">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton
{
	border-style: flat;
	font: 12px;
	color: #DEDEDE;	
}


QToolButton:checked
{
	color: #0DAF9D;
}

</string>
       </property>
       <property name="text">
        <string>3D</string>
       </property>
       <property name="iconSize">
        <size>
         <width>39</width>
         <height>39</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="popupMode">
        <enum>QToolButton::DelayedPopup</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="toolButtonZoomOut">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../Image.qrc">
           <normaloff>:/Resources/ModelButton/zoom_out.png</normaloff>
           <normalon>:/Resources/ModelButton/zoom_out_sel.png</normalon>:/Resources/ModelButton/zoom_out.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>39</width>
           <height>39</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonZoomIn">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../Image.qrc">
           <normaloff>:/Resources/ModelButton/zoom_in.png</normaloff>
           <normalon>:/Resources/ModelButton/zoom_in_sel.png</normalon>:/Resources/ModelButton/zoom_in.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>39</width>
           <height>39</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonNormal">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../Image.qrc">
           <normaloff>:/Resources/ModelButton/default.png</normaloff>
           <normalon>:/Resources/ModelButton/default_sel.png</normalon>:/Resources/ModelButton/default.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>39</width>
           <height>39</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>42</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonLeft">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../Image.qrc">
           <normaloff>:/Resources/ModelButton/right.png</normaloff>
           <normalon>:/Resources/ModelButton/right_sel.png</normalon>:/Resources/ModelButton/right.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>39</width>
           <height>39</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonRight">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../Image.qrc">
           <normaloff>:/Resources/ModelButton/left.png</normaloff>
           <normalon>:/Resources/ModelButton/left_sel.png</normalon>:/Resources/ModelButton/left.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>39</width>
           <height>39</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonForward">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../Image.qrc">
           <normaloff>:/Resources/ModelButton/front.png</normaloff>
           <normalon>:/Resources/ModelButton/front_sel.png</normalon>:/Resources/ModelButton/front.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>39</width>
           <height>39</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonTop">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../Image.qrc">
           <normaloff>:/Resources/ModelButton/top.png</normaloff>
           <normalon>:/Resources/ModelButton/top_sel.png</normalon>:/Resources/ModelButton/top.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>39</width>
           <height>39</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonOblique">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../Image.qrc">
           <normaloff>:/Resources/ModelButton/oblique.png</normaloff>
           <normalon>:/Resources/ModelButton/oblique_sel.png</normalon>:/Resources/ModelButton/oblique.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>39</width>
           <height>39</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="GLWidget" name="openGLWidget">
       <property name="minimumSize">
        <size>
         <width>555</width>
         <height>530</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ImageWidget" name="imageWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>555</width>
         <height>530</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #454449;
</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>22</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetButton">
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>82</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QListWidget{ 
    outline:0px;
}

QListView::item{
	border-radius:5px;
	border: 1px solid #454449;
	margin-top:10px;
	margin-bottom:10px;
	width: 80;
	height:145;
}

QListView::item::hover{
	background-color: #454449;
}

QListView::item:selected{
	background-color: #0DAF9D;
}	
padding-top:10px;
padding-bottom:10px;</string>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="gridSize">
        <size>
         <width>80</width>
         <height>165</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>22</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>30</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>100</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonMeasureStart">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton
{

	border-style: flat;
	font: 12px;
	color: #DEDEDE;
	border: 1px solid #1F2026 ;
	border-radius: 25px;

}

QToolButton:checked
{
	color: #0DAF9D;
}

QToolButton:disabled
{
	color: #24242c;
}
</string>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
       <property name="icon">
        <iconset resource="../../Image.qrc">
         <normaloff>:/Resources/Navigation/play.png</normaloff>
         <normalon>:/Resources/Navigation/play_sel.png</normalon>:/Resources/Navigation/play.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="popupMode">
        <enum>QToolButton::DelayedPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonMeasureSetting">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton
{

	border-style: flat;
	font: 12px;
	color: #DEDEDE;
	border: 1px solid #1F2026 ;
	border-radius: 25px;

}
QToolButton:checked
{
	color: #0DAF9D;
}

QToolButton:disabled
{
	color: #24242c;
}
</string>
       </property>
       <property name="text">
        <string>测量设置</string>
       </property>
       <property name="icon">
        <iconset resource="../../Image.qrc">
         <normaloff>:/Resources/Navigation/measure_setting.png</normaloff>
         <normalon>:/Resources/Navigation/measure_setting_sel.png</normalon>:/Resources/Navigation/measure_setting.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonDeviceSetting">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton
{

	border-style: flat;
	font: 12px;
	color: #DEDEDE;
	border: 1px solid #1F2026 ;
	border-radius: 25px;

}

QToolButton:checked
{
	color: #0DAF9D;
}

QToolButton:disabled
{
	color: #24242c;
}
</string>
       </property>
       <property name="text">
        <string>设备设置</string>
       </property>
       <property name="icon">
        <iconset resource="../../Image.qrc">
         <normaloff>:/Resources/Navigation/device_setting.png</normaloff>
         <normalon>:/Resources/Navigation/device_setting_sel.png</normalon>:/Resources/Navigation/device_setting.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>130</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QWidget</extends>
   <header>UI/Page/ImageWidget/imagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>3D/glwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Image.qrc"/>
 </resources>
 <connections/>
</ui>
